training/training_code/f1.py

```diff
@@ -0,0 +1,220 @@
+# Copyright 2019 The TensorFlow Authors. All Rights Reserved.
```

Comment:
As a general comment if this is the same as what is in `tf.addons`, we should prefer to use the one from `tf.addons`